icgcget/clients/portal_client.py

```diff
@@ -21,56 +21,57 @@
 from icgcget.clients.errors import ApiError
 
 
-def call_api(request, api_url, headers=None, head=False):
+def call_api(request, headers=None, head=False, verify=True):
     logger = logging.getLogger("__log__")
     try:
         if head:
-            resp = requests.head(request, headers=headers)
+            resp = requests.head(request, headers=headers, verify=verify)
         else:
-            resp = requests.get(request, headers=headers)
+            resp = requests.get(request, headers=headers, verify=verify)
     except requests.exceptions.ConnectionError as ex:
```

Comment:
http://stackoverflow.com/questions/6470428/catch-multiple-exceptions-in-one-line-except-block